streams.json

```diff
@@ -25,10 +27,12 @@
           "repository_type": "GIT",
           "repository_url": "https://github.com/aeshell/aesh-readline",
           "codebase": "master",
-          "tag": "1.1",
-          "version": "1.1",
-          "gav": "org.aesh",
-          "comment": ""
+          "tag": "",
+          "group_id": "org.aesh",
+          "artifact_id": "aesh-readline",
+          "version": "1.2",
+          "gav": "aesh-readline:org.aesh:1.2",
+          "comment": "TAG suspicious. TAG in pom: 1.1, TAG in mapping tool: . This component was not found in prod KB, it was inferred from the dictionary. "
```

Comment:
There is no tag in the pom. Not sure what's going on here.

Reply:
The TAG in the pom is 1.1 but mapping tool was not able to retrieve a valid TAG for this component, this looks like a bug in mapper instead, @gaol WDYT?

Reply:
Component: `aesh-readline:org.aesh` is not included in the mapping tool yet, I will add it.

https://gitlab.cee.redhat.com/lgao/jboss-set-mapping/issues/3